.rubocop.yml

```diff
@@ -3,7 +3,6 @@ inherit_gem:
 
 AllCops:
   NewCops: enable
-  TargetRubyVersion: 3.0
```

Comment:
hmm this is slightly wrong, because we want to support ruby 3.0, but prefer 3.3+ in development

Reply:
The TargetRubyVersion should be 3.1 since it's the [oldest supported version](https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version) that the gem supports. RuboCop implicitly reads from required_ruby_version for this value (which is set to >=3.1 at the moment).